pyproject.toml

```diff
@@ -10,4 +10,8 @@ requires = [
     "epicscorelibs>=7.0.7.99.1.1a2; python_version>='3.12'",
     "pvxslibs>=1.3.1",
     "pvxslibs>=1.3.2a1; python_version>='3.12'",
+    "packaging>=24.2",
```

Comment:
What needs this new package? If it's 3.13 only then shouldn't we include the relevant `python_version` specifier?